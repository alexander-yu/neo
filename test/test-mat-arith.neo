void main() {
    create matrix<int> a = [[1, 2], [3, 4]];
    create matrix<int> b = [[1, 2], [3, 4]];
    print(a + b);
    print("\n");
    print(2 * a);
    print("\n");
    create matrix<float> c = [[1., 2.], [3., 4.]];
    print(0.5 ^ c);
    print("\n");
    print(c <= 2.);
    print("\n");
    create matrix<float> d = [[0.5, 0.3]];
    print(c % d);
}