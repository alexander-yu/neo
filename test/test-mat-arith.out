[[2, 4],
 [6, 8]]
[[2, 4],
 [6, 8]]
[[0.5, 0.25],
 [0.125, 0.0625]]
[[1, 1],
 [0, 0]]
[[0, 0.2],
 [0, 0.1]]
